docs/docs/quick_start/quickstart_local_python.md

```diff
@@ -85,3 +85,75 @@ python test_bodo.py
 By default Bodo will use all available cores. To set a limit on the number of processes spawned, set the environment variable `BODO_NUM_WORKERS`.
 Note that the first time you run this code, it may take a few seconds to compile the code.
 Next time you run the code, it will execute much faster. Check the [Python API Reference][pythonreference] for the full list of supported Python operations.
+
+
+## Bodo Integration Quickstart
+
+Here are high level steps for integrating Bodo into Python workloads:
+
+1. Installation and Import
+
+    a. [Install Bodo](../installation_and_setup/install.md) (e.g. `pip install bodo`).
+
+    b. In each file where parallelization is desired, add:
+        ```python
+        import bodo
+        ```
+
+2. JIT-Compile Your Main Processing Functions
+
+    a. Decorate computationally intensive functions with `@bodo.jit(cache=True)`.
+
+    b. Gather all configuration inputs (e.g., file lists) outside of the jitted function and pass those inputs into the jitted function as parameters.
+
+    * Any file discovery logic (e.g., `glob.glob`, or referencing environment variables) should happen _outside_ your jitted function.
+
+    * The jitted function should receive the final list of files/tables and other necessary parameters explicitly.
+
+    c. Keep I/O of large data inside JIT functions if the storage format is supported by Bodo (Parquet, CSV, JSONL, Iceberg, Snowflake).
```

Comment:
```suggestion
    c. Keep I/O of large data inside JIT functions if the storage format is supported by Bodo (Parquet, CSV, JSON, Iceberg, Snowflake).
```

Reply:
JSONL is "JSON lines".

Reply:
In that case, you need to update other mentions of JSON in docs (README, file_io, ...etc) as it's confusing.

Reply:
Changed to JSON for now. We need to decide on next steps of JSON in general and see if we want to limit support to JSONL or at least promote it more.